charts/timber-observation-service/values.yaml

```diff
@@ -0,0 +1,264 @@
+global:
+  # -- Extra pod labels in a map[string]string format, most likely to be used for the costing labels.
+  extraPodLabels: {}
+
+observationService:
+  image:
+    # -- Docker registry for Observation Service image
+    registry: ghcr.io
+    # -- Docker image repository for Observation Service
+    repository: caraml-dev/timber/observation-service
+    # -- Docker image tag for Observation Service
+    tag: v0.0.0-build.15-b8afdb5
+    # -- Docker image pull policy
+    pullPolicy: IfNotPresent
+
+  # -- Annotations to add to Observation Service pod
+  annotations: {}
+
+  replicaCount: 1
+
+  # -- Observation Service server configuration.
+  apiConfig: {}
+    # Default configuration base on caraml-dev/timber/observation-service: v0.0.0-build.15-b8afdb5
```

Comment:
this chunk is extracted from the obs-service code. Ideally we should have a doc on the configurables and ref the url.

Reply:
Noted, can we keep track this action item in JIRA?